tests/test_configurer.py

```diff
@@ -127,20 +126,20 @@ def test_auth_dummy():
     assert c.DummyAuthenticator.password == 'test'
 
 
-def test_auth_firstuse():
+def test_auth_nativeauth():
```

Comment:
I think this test should be in addition to the firstuse test, not replace it. We'll stil ship first use authenticator.